tools/internal_ci/linux/grpc_xds_k8s_lb.sh

```diff
@@ -180,6 +180,9 @@ main() {
     "remove_neg_test"
     "round_robin_test"
   )
+  if [[ "${TESTING_VERSION}" == "master" ]]; then
```

Comment:
This works now we merged the fix. Switch this == to ~=.

Reply:
Alright, done